acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/DrivingLicenceAPIPage.java

```diff
@@ -59,13 +74,123 @@ public void postRequestToSessionEndpoint() throws IOException, InterruptedExcept
+    }
+
+    public void retryValueInDLCheckResponse(String retry) {
+        assertTrue(StringUtils.isNotBlank(RETRY));
+        Assert.assertEquals(retry, RETRY);
+    }
+
+    public void getAuthorisationCodeforDL() throws IOException, InterruptedException {
+        String privateApiGatewayUrl = configurationService.getPrivateAPIEndpoint();
+        String coreStubUrl = configurationService.getCoreStubUrl(false);
+
+        HttpRequest request =
+                HttpRequest.newBuilder()
+                        .uri(
+                                URI.create(
+                                        privateApiGatewayUrl
+                                                + "/authorization?redirect_uri="
+                                                + coreStubUrl
+                                                + "/callback&state="
+                                                + STATE
+                                                + "&scope=openid&response_type=code&client_id=ipv-core-stub"))
+                        .setHeader("Accept", "application/json")
+                        .setHeader("Content-Type", "application/json")
+                        .setHeader("session-id", SESSION_ID)
+                        .GET()
+                        .build();
+        String authCallResponse = sendHttpRequest(request).body();
+        LOGGER.info("authCallResponse = " + authCallResponse);
+        AuthorisationResponse deserialisedResponse =
+                objectMapper.readValue(authCallResponse, AuthorisationResponse.class);
+        AUTHCODE = deserialisedResponse.getAuthorizationCode().getValue();
+        LOGGER.info("authorizationCode = " + AUTHCODE);
+    }
+
+    public void postRequestToAccessTokenEndpointForDL(String criId)
+            throws IOException, InterruptedException {
+        String accessTokenRequestBody = getAccessTokenRequest(criId);
+        LOGGER.info("Access Token Request Body = " + accessTokenRequestBody);
+        String publicApiGatewayUrl = configurationService.getPublicAPIEndpoint();
+        LOGGER.info("getPublicAPIEndpoint() ==> " + publicApiGatewayUrl);
+        HttpRequest request =
+                HttpRequest.newBuilder()
+                        .uri(URI.create(publicApiGatewayUrl + "/token"))
+                        .setHeader("Accept", "application/json")
+                        .setHeader("Content-Type", "application/json")
+                        .POST(HttpRequest.BodyPublishers.ofString(accessTokenRequestBody))
+                        .build();
+        String accessTokenPostCallResponse = sendHttpRequest(request).body();
+        LOGGER.info("accessTokenPostCallResponse = " + accessTokenPostCallResponse);
+        Map<String, String> deserialisedResponse =
+                objectMapper.readValue(accessTokenPostCallResponse, new TypeReference<>() {});
+        ACCESS_TOKEN = deserialisedResponse.get("access_token");
+    }
+
+    public String requestDrivingLicenceCRIVC()
```

Comment:
Can we keep naming consistent we've name the other methods post but this request

Reply:
Yes, have now renamed the method to postRequestToDrivingLicenceVCEndpoint()